core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -70,7 +70,7 @@ public class InstancedConfiguration implements AlluxioConfiguration {
    * @return an instanced configuration preset with defaults
    */
   public static InstancedConfiguration defaults() {
```

Comment:
emmmm, this defaults() may be confusing for end users, they may not expect a copy to happen similar to ConfigurationUtils.defaults(）before

Reply:
Ya, that's the problem. The logic didn't change. I just renamed the function. I had an earlier PR to refactor InstancedConfiguration. But I figured we should fix the static part first.